### PR TITLE
Final check before v1.3 closes and UG updates

### LINKED ISSUE #138: Update user guide
One of week 11's tp task

### PR DESCRIPTION
**Pull request by tengcharmaine**
Checked all the commands and all seems to be working okay. No issues spotted.

Corrected typos in UG, updated/added images to UG and added more FAQs.

Let me know if there are any typos/issues!

Closes #138.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -158,19 +160,22 @@ Adds a person to ClinicMate.
 Format: `add n/NAME p/PHONE e/EMAIL i/IC_NUMBER ag/AGE s/SEX a/ADDRESS`
 
 * A person will be uniquely identified by his/her personal identification number.
-* Clinic mate does not allow the same identification number to be twice.
+* ClinicMate does not allow the same identification number to be twice.
```
Comment (drustanyjt) [inline docs/UserGuide.md]: type here i think you meant used twice.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -249,23 +263,33 @@ If your changes to the data file makes its format invalid, ClinicMate will disca
...
 
-### Archiving data files `[coming in v2.0]`
-
-_Details coming soon ..._
-
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
+**Q**: When using multiple screens, moving the application to a secondary screen and later switch to using only the primary screen, the GUI opens off-screen. What do I do?<br>
```
Comment (drustanyjt) [inline docs/UserGuide.md]: slight grammar mistake "later switching"

**Review by drustanyjt — APPROVED** [review]
There's some typos. But I think we're going to edit the UG today so maybe we can just merge first then edit those as we update the UG.

File: docs/UserGuide.md (hunk 2 above)
Comment (tengcharmaine) [reply docs/UserGuide.md]: changed

File: docs/UserGuide.md (hunk 1 above)
Comment (tengcharmaine) [reply docs/UserGuide.md]: changed

**Pull request merged**